SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/DCPower/Source.cs

```diff
@@ -484,7 +484,7 @@ public static void ForceCurrentAsymmetricLimit(this DCPowerSessionsBundle sessio
         /// <param name="sequenceLoopCount">The number of times to force the sequence.</param>
         /// <param name="transientResponse">The transient response to use.</param>
         /// <param name="sequenceTimeoutInSeconds">The maximum time used to force the sequence.</param>
-        internal static void ForceVoltageSequenceSynchronized(
+        public static void ForceVoltageSequenceSynchronized(
```

Comment:
@lornazh, I am hesitant to expose this one if we do not have a unit test that confirms it at least runs (not necessarily to provide it's synchronized, I know that is hard).

Reply:
I hesitated on that one too. Changed back to internal.